addons/mass_mailing_marketing_card/views/snippets/snippets.xml

```diff
@@ -0,0 +1,86 @@
+<odoo>
+<template id="email_designer_snippets" inherit_id="mass_mailing.email_designer_snippets">
+    <xpath expr="//t[@id='default_snippets']//snippets[@id='email_designer_marketing_elements']" position="inside">
+        <t t-snippet="mass_mailing_marketing_card.s_call_to_share_card" string="Share a Card" t-thumbnail="/mass_mailing_marketing_card/static/src/img/snippets_thumbs/s_call_to_share_card.svg"/>
+    </xpath>
+</template>
+
+<template id="web_editor_snippet_options_inherit" inherit_id="web_editor.snippet_options">
+    <!--Apply image resize to the anchor instead-->
+    <xpath expr="div[@data-js='ImageTools']" position="attributes">
+        <attribute name="data-exclude" add=".s_call_to_share_card img" separator=", "/>
+    </xpath>
+    <xpath expr="div[@data-js='ImageTools']" position="after">
+        <div data-js="ImageTools"
+         data-selector=".s_call_to_share_card img">
+            <we-input string="Description" class="o_we_large"
+            data-select-attribute="" data-attribute-name="alt"
+            placeholder="Alt tag"
+            title="'Alt tag' specifies an alternate text for an image, if the image cannot be displayed (slow connection, missing image, screen reader ...)."/>
+            <we-input string="Tooltip" class="o_we_large"
+                data-select-attribute="" data-attribute-name="title"
+                placeholder="Title tag"
+                title="'Title tag' is shown as a tooltip when you hover the picture."/>
+        </div>
+    </xpath>
+</template>
+<template id="mass_mailing_snippet_options_inherit" inherit_id="mass_mailing.snippet_options">
+    <!--We don't want users to edit the image and href-->
+    <xpath expr="div[@data-js='ReplaceMedia']" position="attributes">
+        <attribute name="data-exclude" add=".s_call_to_share_card img" separator=", "/>
+    </xpath>
+
+    <!--Alignment doesn't work well for images inside anchors-->
+    <!--TODO fix for all anchored images-->
+    <xpath expr="div[.//we-select[@id='o_we_image_alignment_option']]" position="attributes">
+        <attribute name="data-exclude" add=".s_call_to_share_card img" separator=", "/>
+    </xpath>
+    <xpath expr="div[.//we-select[@id='o_we_image_alignment_option']]" position="after">
```

Comment:
I don't understand the difference between doing this with a xpath or putting the option directly inside the MarketingCard snippet here below?

EDIT oh, I think I get it, it's to have those options in the appropriate sections.

Reply:
Yes I have to add "data-exclude" for elements inside the card so people can't modify them "normally" and break the whole layout (less than they can now at least).